### PR TITLE
ci(v2): test swizzle on CI using v2 website

### PR DESCRIPTION
**Pull request by Simek**
## Motivation

#3830 - as reported, there is an issue with swizzled component form `theme-classic` right now. 

To prevent new swizzling errors from slipping into the release I have decided to add the swizzle test of most of `theme-classic`, `theme-search-algolia` and `theme-live-codeblock` components to the CI pipeline. Test is based on the Docusuaurs V2 website and helper scripts starting with `swizzle:v2` have been added to the main `package.json`. 

The list of components is subjective, feel free to change or update it 🙂 

Currently the test step will fail for:
 * `DocPage` and `DocSidebar` components due to the same error as reported in the mentioned issue
   > ModuleNotFoundError: Module not found: Error: Can't resolve '../icons/arrow.svg'

After fixing those problem the step run should be successful.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

Manual command run and test builds on local. After publish

## Related PRs

No.

### CODE REVIEW

**Comment by slorber** [comment]
Thanks, that's a good test idea 👍 

We should probably ensure the test pass before merging 😅

File: .github/workflows/swizzle-test.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['12']
+    steps:
+      - uses: actions/checkout@v1
```
Comment (HonkingGoose) [inline .github/workflows/swizzle-test.yml:15]:
```suggestion
      - uses: actions/checkout@v2
```

`actions/checkout` has a new stable release at `v2`: https://github.com/actions/checkout/releases/tag/v2.0.0
Latests stable is `v2.3.4`, but the `v2` tag points to latest `v2` release as well.

File: .github/workflows/swizzle-test.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      - name: Use Node.js ${{ matrix.node }}
+        uses: actions/setup-node@v1
```
Comment (HonkingGoose) [inline .github/workflows/swizzle-test.yml:17]:
```suggestion
        uses: actions/setup-node@v2
```

`actions/setup-node` also has a newer stable release `v2`: https://github.com/actions/setup-node/releases/tag/v2.1.4

**Comment by Simek** [comment]
FYI: From the day when #3854 has been merged (Dec 01, 2020), running all the commands locally, in same order as in workflow results in successful build. The server compilation seems to fail only on the CI. I have tried many ideas to fix this, all of them failed. 

To be honest, I don't want to waste any minute more on this problem, @slorber can you step in?

**Pull request closed without merging**